txs/mempool_iterator.go

```diff
@@ -70,8 +70,8 @@ func (pq *priorityQueue) update(it *item, ntx *NanoTX) {
 	heap.Fix(pq, it.index)
 }
 
-// mempoolIterator holds the best transaction from the conservative state mempool.
-// not thread-safe.
+// MempoolIterator holds the best transaction from the conservative state mempool.
```

Comment:
Minor nit: comment should start with type name, so it should not be capitalized there.

Reply:
Thanks 🙂 That's a leftover from the `godot` linter that had stricter configuration by default then now (forcing all comments to start capitalized). I disabled it and probably missed reverting this.